### DISCUSSION TITLE
Use line-width to signal wikidata availability

### DISCUSSION THREAD

**Discussion by Sequynth** [EqualStreetNames/equalstreetnames#768] [Ideas]
Currently, line-width and line-color are used simultaneously to indicate that the gender of the etymological figure for a streetname is known. I'd like to suggest the independent use of line-color and line-width:

line-color: dependent on the gender; gray for non-human etymology. (No change from current behaviour)

line-width: thin width when no wikidata is available; larger width, when wikidata is available.

I think this makes the websites response more predictable from the visitors point of view. It will also allow users to find streets where they can learn more about the person behind the streetname. Using it on non-human related streetnames seems like a nice addition, since people on the website might actually be interested in the etymology of other streets.

**Comment by samwilson** [comment]
This sounds like a great idea. I'm using equalstreetnames as a bit of a todo list for which Wikiata items need their etymology researched. But it's hard to see the difference between non-human and no-named-after.
